experimenter/experimenter/experiments/models.py

```diff
@@ -1240,6 +1240,20 @@ def recipe_json(self):
             .replace("\\n", "\n")  # Handle hard coded newlines in targeting
         )
 
+    @property
+    def draft_date(self):
+        first_changelog_entry = self.changes.order_by("changed_on").first()
+        return first_changelog_entry.changed_on.date() if first_changelog_entry else None
+
+    @property
+    def preview_date(self):
+        preview_changelog = (
+            self.changes.filter(new_status=self.Status.PREVIEW)
+            .order_by("changed_on")
+            .first()
```

Comment:
Should this be first or last? If it changes out of preview and back into preview would we want to show the first or most recent date?

Reply:
(and same question for `draft_date`)

Reply:
yes I had the same doubt and thought to use the first one, but we can change depending who users want to see it